eslint.config.mjs

```
@@ -0,0 +1,80 @@
+        'error',
+        {
+          skipStrings: true,
+          skipComments: true,
+          skipRegExps: true,
+          skipTemplates: true,
+        },
+      ],
+      '@stylistic/js/indent': ['error', 4],
+    }
+  },
+  // {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(@stylistic/js/indent)